### DISCUSSION TITLE
Change opacity on dim-inactive color when using transparent

### DISCUSSION THREAD

**Discussion by Esaiy** [EdenEast/nightfox.nvim#342] [Q&A]
I want all windows to be transparent but also make the focused window different from the other. Using dim-inactive makes the other window uses the bg0 and no longer makes it transparent. I tried changing the bg0 in palettes and inactive in specs value with rbga value and color lib but both didn't work. Is there some way to make this work?

**Comment by vulbyte** [comment]
+1'ing this because the carbon fox theme is almost perfect, but i just want the opacity of the unfocused window to be like 0.2 but i can't for the life of me seem to get it to work

**Comment by serranomorante** [comment]
I'm using `levouh/tint.nvim` to achieve that effect.

[.dotfiles](https://github.com/serranomorante/.dotfiles)
